subprojects/core/src/test/groovy/org/gradle/internal/operations/notify/BuildOperationNotificationBridgeTest.groovy

```diff
@@ -61,243 +54,17 @@ class BuildOperationNotificationBridgeTest extends Specification {
         }
     }
 
-    def "does not allow duplicate registration"() {
```

Comment:
Looks like we deleted too much from this test.

Reply:
re-added them. Thanks